docs/ReferenceAssemblies.md

```diff
@@ -0,0 +1,25 @@
+Reference assemblies
+========================
+
+Most projects in this repo have a `ref` directory next to their `src` that contains the project and source code for a reference assembly.
```

Comment:
Suggest "Some" or "Many" instead of "Most". I can only find one `ref/` directory at the moment.

Reply:
> I can only find one `ref/` directory at the moment.

What do you mean? At least a hundred was added.

Reply:
I hadn't seen that when I made that comment. Should have deleted it.